# Erste Exception

try:
	#open("XXX.txt")
	print(mun)
	print("------1------")
except NameError:
	print("Reaktion, wenn ein Exception gefunden wird")

print("------2------")
